lib/mongo/session.ex

```diff
@@ -339,9 +339,10 @@ defmodule Mongo.Session do
   @impl :gen_statem
   # Abort all pending transactions if there any and end session itself.
   def terminate(_reason, state, %{pid: pid} = data) do
-    if state == :in_transaction do
-      _ = try_run_txn_command(data, :abortTransaction)
-    end
+    _ =
+      if state == :in_transaction do
+        _ = try_run_txn_command(data, :abortTransaction)
+      end
```

Comment:
```suggestion
    _ = if state == :in_transaction, do: try_run_txn_command(data, :abortTransaction)
```

I think that this will be clearer. Alternatively we can keep the multiline, but remove `_ =` within the `if` as this is pointless repetition.

Reply:
I kept the the multiline style and removed  unnecesary assignment  within  if block as per suggested by you.